### ISSUE TITLE
Jailer of Temperance seems to be despawning on spawn.

### ISSUE THREAD

**Issue by PikaFox** [LegionDark/Issues#319]
**Date & Time**: 9/18/15
**Frequency**: Always
**FFXI Client Version**: 30150730_1
**Character Name**: Tarot
**Race**: Hume
**Nation**: San
**Main Job**: Thf
**Sub Job**: Dnc
**Area & Coordinates**: Grand Palace of Hu'Xzoi H-4
**Party or Solo**: NA
**NPC or Monster Name**: Jailer of Temperance  16916814
**ffxiah.com link (for inventory items only)**:  NA
**Multi-boxing? (multiple clients on same connection)**: no
**Steps To Reproduce The Issue**:

Maiku camped the mob for a few hours. Mob is supposed to be a 15 min repop semi lotto. He hasnt seen the mob since. I looked, couldnt find the mob, so i tried to bring it to me. It instantly depopped. Tried to spawn the mob, found it spawns in the room at the top at H-4, and it autodespawned every attempt to spawn. He shouldnt be a force pop, so he shouldnt be despawning on spawn without enmity.

**Comment by TeoTwawki** [comment]
This tends to happen with timed pop NM when we use GM commands to force them. Its more likely the spawn is broken altogether than it instantly depopping under normal conditions.

**Comment by Roamin** [comment]
I am currently fighting Jailer of Temperance, and it spawned just like retail.  I had tested this a few days prior with maiku , and we figured out that it was spawned like retail, by killing one of the 5 place holders.

I went around and killed them, out of the 5 , the south-west one was not up , and i am alone in zone so that's odd, but killing the one in south east tower had jailor of temperance pop.
